sdk/storage/azure-storage-queue/azure/storage/queue/_shared/authentication.py

```diff
@@ -45,7 +45,7 @@ def _wrap_exception(ex, desired_type):
     # in general, or that will not result in an error in 3
     # However, we can keep the previous error type and message
     # TODO: In the future we will log the trace
-    return desired_type('{}: {}'.format(ex.__class__.__name__, msg))
+    return desired_type(f'{ex.__class__.__name__}: {msg}')
```

Comment:
Anything in the `_shared` folder is duplicated across all Storage packages (blob, files-datalake, files-share, and queue) so if we make changes for one package, we need to make them for all four packages.

You can either revert changes in `_shared` and we'll handle that later or update the `_shared` folder in other packages in this PR as well.